wandb/__init__.pyi

```diff
@@ -155,40 +155,40 @@ def setup(settings: Settings | None = None) -> _WandbSetup:
             overridden by subsequent `wandb.init()` calls.
 
     Example:
-        ```python
-        import multiprocessing
+    ```python
```

Comment:
Should the language be specified? I thought the default in Python docstrings is assumed to be Python?

Reply:
We need it for the markdown files that are autogenerated.